nautobot/core/templates/generic/object_list.html

```diff
@@ -3,12 +3,29 @@
 {% load helpers %}
 {% load static %}
 
+
+{% block extra_styles %}
+<style>
+    .btn-filter-value {
+        background: #cfcfcf;
+        margin-right: 5px;
+    }
+    .filters-applied>.btn {
+        margin-right: 5px;
+        margin-bottom: 5px;
+    }
+</style>
```

Comment:
Is this still relevant/needed? I don't see `btn-filter-value` or any `btn` inside of `.filters-applied` in the current HTML.